crap-advisor-frontend/src/utils/index.ts

```diff
@@ -1,25 +1,35 @@
-import {Location, Primitive} from "../app/typings";
-import {isNil} from 'lodash';
-import {LatLngExpression} from "leaflet";
+import { LatLngExpression } from 'leaflet';
+import { isNil } from 'lodash';
 
-export const params2form = (
-    payload: Record<string, Primitive | Primitive[]>
-): string =>
-    Object.keys(payload)
-        .filter((key) => !isNil(payload[key]))
-        .map((key) => {
-            if (Array.isArray(payload[key])) {
-                return (payload[key] as Primitive[])
-                    .filter((item) => !isNil(item))
-                    .map(
-                        (item) => `${encodeURIComponent(key)}=${encodeURIComponent(item)}`
-                    )
-                    .join('&');
-            }
-            return `${encodeURIComponent(key)}=${encodeURIComponent(
-                payload[key] as Primitive
-            )}`;
-        })
-        .join('&');
+import { Location, Primitive } from 'app/typings';
 
-export const toLatLngExpression = (location: Location): LatLngExpression => [location.latitude, location.longitude]
\ No newline at end of file
+export const params2form = (payload: Record<string, Primitive | Primitive[]>): string =>
```

Comment:
the more i look at this function the worse it becomes...

Reply:
Is something WRONG with it?? Okay nerd, say it to akbars developer face. He's tryina his best, you douchebag.

Reply:
@aiecel Get my apologize, buddy. I shouldn't have insulted you. My bad.

Reply:
your social credit just decreased by 100 points